outline/tunnel.go

```diff
@@ -97,6 +98,8 @@ func (t *outlinetunnel) registerConnectionHandlers() {
 	} else {
 		udpHandler = dnsfallback.NewUDPHandler()
 	}
-	core.RegisterTCPConnHandler(oss.NewTCPHandler(t.host, t.port, t.password, t.cipher))
+	tcpHandler := oss.NewTCPHandler(t.host, t.port, t.password, t.cipher)
```

Comment:
This would read and scale better if we created the ss Client here, than pass it to the `TCPConnHandler`. Thoughts? Can you make that change?

Same for the Electron code.

Reply:
OK, I've written that up in #96.  Let's merge that one first.

Reply:
Great! I approved the other PR, let me know when I should take another look here.

Reply:
OK, this is ready for review.